prototypes/form-wizard/_plugins/jekyll-agency-model-builder.rb

```diff
@@ -0,0 +1,53 @@
+require 'pp'
+require 'json'
+
+module Jekyll
+  class AgencyJsonGenerator < Generator
+    safe true
+
+    DEFAULT_YAML_DIR = __dir__ + '/../../../../2015-foia/contacts/data'
```

Comment:
:+1: maybe we want to mention this in the README, that you want to clone the directory next to `foia-recommendations`, right?

Or that you can override it via env var.